app/assets/javascripts/modules/external_users/claims/fee_calculator/FeeCalculator.UnitPrice.js

```diff
@@ -172,13 +172,13 @@
       // only some errors will have a JSON response
       this.clearErrors(context);
       var $label = $(context).find('label');
-      var $calculated = $(context).closest('.fx-fee-group').find('.js-fee-calculator-success').find('input');
+      var $price_calculated = $(context).closest('.fx-fee-group').find('.js-fee-calculator-success').find('input');
```

Comment:
I believe context is now the `.fx-fee-group` so could refactor
```
var $price_calculated = $(context).find('.js-fee-calculator-success > input');
```

Reply:
👍